<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CatalogsDBUI</class>
 <widget class="QDialog" name="CatalogsDBUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>783</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DSO Catalogs</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTableWidget" name="objectsTable">
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="importButton">
         <property name="text">
          <string>Import Catalog...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="createButton">
         <property name="text">
          <string>Create Catalog...</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="activateButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Enable</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="removeButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Remove</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="exportButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Export...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="dublicateButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Clone...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="colorButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Colors</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="moreButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>More...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CatalogsDBUI</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CatalogsDBUI</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
